mst/vpc-peering/vpc-peering-azure.md

```diff
@@ -0,0 +1,264 @@
+    resource names are case-agnostic, but the Aiven API currently only accepts
+    names in lower case. If no error is shown, the peering connection is being set
+    up by the Timescale platform.
+
+1.  Run the following command until the state is no longer `APPROVED` , but
+    `PENDING_PEER`:
+
+    ```bash
+    avn vpc peering-connection get -v --project-vpc-id $aiven_project_vpc_id --peer-cloud-account $user_subscription_id --peer-resource-group $user_resource_group --peer-vpc $user_vnet_name
+    ```
+
+    A state such as `INVALID_SPECIFICATION`  or `REJECTED_BY_PEER`  may be shown
+    if the VNet specified did not exist, or the Timescale app object wasn't
+    given permissions to peer with it. If that occurs, check your configuration
+    and then recreate the peering connection. If everything went as expected,
+    the state changes to `PENDING_PEER`  within a couple of minutes showing
+    details to set up the peering connection from your VNet to the Project VPC's
+    VNet in the Timescale subscription.
+
+    Save the `to-tenant-id` field in the output. It is referred to as the
+    `aiven_tenant_id`. The `to-network-id`  field from the output is referred to
+    as the `$aiven_vnet_id`.
+
+1.  Log out the Azure user you logged in using:
+
+    ```bash
+    az account clear
+    ```
+
+1.  Log in the application object you created to your AD tenant using:
+
+    ```bash
+    az login --service-principal -u $user_app_id -p $user_app_secret --tenant $user_tenant_id
+    ```
+
+1.  Log in the same application object to the Timescale AD tenant:
+
+    ```bash
+    az login --service-principal -u $user_app_id -p $user_app_secret --tenant
+    $aiven_tenant_id
+    ```
+
+    Now your application object has a session with both AD tenants
+
+1.  Create a peering from your VNet to the VNet in the Timescale subscription:
+
+    ```bash
+    az network vnet peering create --name <peering name of your choosing> --remote-vnet $aiven_vnet_id --vnet-name $user_vnet_name --resource-group $user_resource_group --subscription $user_subscription_id --allow-vnet-access
+    ```
+
+    If you do not specify `--allow-vnet-access` no traffic is allowed to flow
+    from the peered VNet and Timescale services cannot be reached through the
+    peering. After the peering has been created the peering should be in state
+    connected.
+
+    In case you get this error, it's possible the role assignment hasn't taken
+    effect yet. If that is the case, try logging in again and creating the
+    peering again after waiting a bit by repeating the commands in this step. If
+    the error message persists, check the role assignment was correct.
+
+    The client `<random uuid>` with object id `<another random uuid>` does not have
+    authorization to perform action
+    `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write` over scope
+    '$user_vnet_id' If access was recently granted, refresh your credentials.
```

Comment:
Maybe put this inside a code block to make it clearer that it's an error message

Reply:
Thanks @charislam !!! fixed